x-pack/test/functional/page_objects/gis_page.js

```diff
@@ -132,8 +132,9 @@ export function GisPageProvider({ getService, getPageObjects }) {
     async openNewMap() {
       log.debug(`Open new Map`);
 
-      await this.gotoMapListingPage();
-      await testSubjects.click('newMapLink');
+      // Navigate directly because we don't need to go through the map listing
+      // page. The listing page is skipped if there are no saved objects
+      await PageObjects.common.navigateToUrlWithBrowserHistory(APP_ID, '/map');
```

Comment:
@elastic/kibana-gis This is a new change which I introduced because the functional tests were path-dependent: they assumed that the user had loaded saved maps, and this was not the case in all of the existing tests. This change makes fewer assumptions.